core/models/configuration.rb

```diff
@@ -119,6 +121,12 @@ def docker_configuration?
     !@docker_configuration.empty?
   end
 
+  # Check that configuration is AWS Terraform foncfiguration
+  # @return [Boolean] true if the configuration is AWS Terraform
+  def aws_terraform_configuration?
```

Comment:
I currently do not see why you bind terraform to AWS only. In the future we want to be able to switch to different providers.
Would it be hard to maintain up command and destroy command semantics inspite of the service provider type?